Packs/OTSecurity/IndicatorTypes/MAC_Address.json

```diff
@@ -0,0 +1,12 @@
+{
+	"regex": "^([0-9A-Fa-f]{2}[:-]){5}([0-9A-Fa-f]{2})$",
+	"details": "MAC Address",
+	"disabled": false,
+    "shouldPublish": true,
+    "shouldCommit": true,
+    "expiration": 0,
+    "version": -1,
+    "fromVersion": "6.0.0",
+    "system": false,
+	"id": "MAC Address"
+}
```

Comment:
I think that indicator type should be associated to layout

Reply:
@evisochek I added a MAC Address layout